src/utils/helpers/application/buildQueryVariables.ts

```diff
@@ -0,0 +1,35 @@
+import { ApplicationListsOrderBy } from '../../../utils/generated/graphql'
+
+export default function buildSortFields(sortString: string) {
+  const sortFields = sortString.split(',')
+  return sortFields.map((field) => getGraphQLSortName(field))
+}
+
+const getGraphQLSortName = (field: string) => {
+  const [fieldName, direction] = field.split(':')
+  return (mapSortFields[fieldName] +
+    (direction === 'asc' ? 'ASC' : 'DESC')) as ApplicationListsOrderBy
+}
+
+const mapSortFields: any = {
+  type: 'TEMPLATE_NAME_',
+  serial: 'SERIAL_',
+  name: 'NAME_',
+  code: 'TEMPLATE_CODE_',
+  applicant: 'APPLICANT_',
+  username: 'APPLICANT_USERNAME_',
+  org: 'ORG_NAME_',
+  stage: 'STAGE_',
+  status: 'STATUS_',
+  outcome: 'OUTCOME_',
+  lastActiveDate: 'LAST_ACTIVE_DATE_',
+}
+
+type PaginationValues = {
+  numberToFetch: number
+  paginationOffset: number
+}
```

Comment:
Why not use `interface` and better to add to the top?

Reply:
Hmm, I feel like this file contains two different "sections" -- the Sorting processing, and the Pagination process, so I felt it was better to keep the interface in its correct section. (You'd probably put Pagination in its own file, but I couldn't bring myself to make an 8-line file. 😜 )